src/MultiSearchPrompt.php

```diff
@@ -132,6 +134,27 @@ public function visible(): array
         return array_slice($this->matches(), $this->firstVisible, $this->scroll, preserve_keys: true);
     }
 
+    /**
+     * Toggle all options.
+     */
+    protected function toggleAll(): void
+    {
+        $allMatchesSelected = collect($this->matches)->every(fn ($label, $key) => $this->isList()
+            ? array_key_exists($label, $this->values)
+            : array_key_exists($key, $this->values));
+
+        if ($allMatchesSelected) {
+            $this->values = array_filter($this->values, fn ($value) => $this->isList()
+                ? ! array_key_exists($value, $this->matches)
```

Comment:
The "unselect all" behaviour doesn't seem to work when using a list. 

I _think_ this fixes it:

```suggestion
                ? ! in_array($value, $this->matches)
```

Reply:
Whoops! That seems to fix it for me, have pushed it up in a commit. Thank you!